pandas/io/common.py

```diff
@@ -232,13 +233,45 @@ def file_path_to_url(path):
 }
 
 
+def _get_compression_method(compression: Union[str, Dict, None]):
+    """
+    Simplifies a compression argument to a compression method string and
+    a dict containing additional arguments.
+
+    Parameters
+    ----------
+    compression : str or dict
+        If string, specifies the compression method. If dict, value at key
+        'method' specifies compression method.
+
+    Returns
+    -------
+    tuple of ({compression method}, any
+              {compression arguments}, dict)
+
+    Raises
+    ------
+    ValueError on dict missing 'method' key
+    """
+    compression_args = {}  # type: Dict[str, Any]
```

Comment:
Is this even necessary? Think creation in the conditional should suffice

Reply:
@WillAyd I suppose it can be put in an else statement. I'll make that change.